### DISCUSSION TITLE
直接点击动态中的视频，只会直接新开当前界面

### DISCUSSION THREAD

**Discussion by wjn04** [BewlyBewly/BewlyBewly#542] [Q&A]
如图：

![屏幕截图 2024-04-12 233343](https://github.com/hakadao/BewlyBewly/assets/117508966/87fe7080-889d-4f21-80c2-427e1e605759)

在主页点击后会直接新主页，在其他视频播放界面点击只是刷新界面

**Comment by hakadao** [comment]
v0.16.4已經修復，edge由於審核速度太慢導致遲遲沒有更新，chrome現在已經是最新的
